besu/src/test/java/org/hyperledger/besu/cli/options/EthProtocolOptionsTest.java

```diff
@@ -116,24 +116,18 @@ public void parsesInvalidEwpMaxGetNodeDataOptionsShouldFail() {
 
   @Override
   EthProtocolConfiguration createDefaultDomainObject() {
-    return EthProtocolConfiguration.builder().build();
+    return EthProtocolConfiguration.defaultConfig();
   }
 
   @Override
   EthProtocolConfiguration createCustomizedDomainObject() {
-    return EthProtocolConfiguration.builder()
-        .maxGetBlockHeaders(
-            PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_BLOCK_HEADERS + 2))
-        .maxGetBlockBodies(
-            PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_BLOCK_BODIES + 2))
-        .maxGetReceipts(
-            PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_RECEIPTS + 2))
-        .maxGetNodeData(
-            PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_NODE_DATA + 2))
-        .maxGetPooledTransactions(
-            PositiveNumber.fromInt(
-                EthProtocolConfiguration.DEFAULT_MAX_GET_POOLED_TRANSACTIONS + 2))
-        .eth65Enabled(!EthProtocolConfiguration.DEFAULT_ETH_65_ENABLED)
+    return ImmutableEthProtocolConfiguration.builder()
```

Comment:
q: when you cmd-click on "ImmutableEthProtocolConfiguration" - does it go to the interface? Or the generated class?
What about "builder()"?

Reply:
It goes to the generated class, in both cases.